### PR TITLE
[ADDED] Incident in Des Moines, Iowa for #806

### PR DESCRIPTION
**Pull request by amardeshbd**
Incident record for reported issue https://github.com/2020PB/police-brutality/issues/806

### CODE REVIEW

File: reports/Iowa.md [hunk 1]
```diff
@@ -99,6 +99,24 @@ id: ia-desmoines-5
 * https://www.facebook.com/permalink.php?story_fbid=1182620968746799&id=100009967212544
 
 
+### Police arrest 17 people during a peaceful protest for Black Lives Matter at the State Capitol  | July 1st
+
+Black Lives Matter advocates gathered at the Capitol to lobby for the governor to sign an executive order restoring felon voting rights by Saturday, July 4.
+
+White protestors were inside the Capitol, police identified some of them claimed to have vandalized police car in the past, and arrests were made inside the Capitol for those individuals. However, the arrested individuals declined the allegations. The matter got worse when protestors followed the police outside the Capitol, and one of the protestors blocked their path and later the crowd tried to pull the arrestees from the custody of the police officers. That is when more crowds and police started to get involved. Police used pepper spray to help disperse the crowd and arrested 17 people, including two juveniles.
```
Comment (emarcey) [inline reports/Iowa.md:106]: I'm not sure "The matter got worse" is the most objective language. I think you could just cut that clause

File: reports/Iowa.md [hunk 2]
```diff
@@ -99,6 +99,24 @@ id: ia-desmoines-5
...
+
+tags: spray, pepper-spray, tackle, beat, arrest
```
Comment (emarcey) [inline reports/Iowa.md:108]: If minors were arrested, please include the "child" tag

**Comment by amardeshbd** [comment]
Can somebody re-run the job? The GitHub CI choked on checkout flow

```
The process '/usr/bin/git' failed with exit code 1
```

**Comment by emarcey** [comment]
The action doesn't work with forks atm

**Comment by amardeshbd** [comment]
> The action doesn't work with forks atm

Good to know. Thanks @emarcey

**Pull request merged**